src/NUnitFramework/framework/Constraints/MsgUtils.cs

```diff
@@ -52,6 +52,11 @@ namespace NUnit.Framework.Constraints
     /// </summary>
     internal static class MsgUtils
     {
+        /// <summary>
+        /// Default amount of items used by <see cref="FormatCollection"/> method.
+        /// </summary>
+        private const int DefaultMaxItems = 10;
```

Comment:
Changes to this file cherry-picked from #3505 via https://github.com/nunit/nunit/pull/3505/commits/dda80d6822f15909374e3cd9e0c11e54f66be9c5